From 37d126b7f720378b8f084b528579fbc5ca18d7c7 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:22:33 -0400
Subject: [cpufreq] rename __cpufreq_set_policy() as cpufreq_set_policy()

Message-id: <1409235879-1283-60-git-send-email-prarit@redhat.com>
Patchwork-id: 89515
O-Subject: [RHEL7.1 PATCH BZ 1134639 059/185] cpufreq: rename __cpufreq_set_policy() as cpufreq_set_policy()
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 037ce8397d23b2f84ccfb879cf4b43277b0454e3
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Wed Oct 2 14:13:16 2013 +0530

    cpufreq: rename __cpufreq_set_policy() as cpufreq_set_policy()

    Earlier there used to be two functions named __cpufreq_set_policy() and
    cpufreq_set_policy(), but now we only have a single routine lets name it
    cpufreq_set_policy() instead of __cpufreq_set_policy().

    This also removes some invalid comments or fixes some incorrect comments.

    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index 8c2a542..2722b54 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -412,7 +412,7 @@ show_one(scaling_min_freq, min);
 show_one(scaling_max_freq, max);
 show_one(scaling_cur_freq, cur);
 
-static int __cpufreq_set_policy(struct cpufreq_policy *policy,
+static int cpufreq_set_policy(struct cpufreq_policy *policy,
     struct cpufreq_policy *new_policy);
 
 /**
@@ -433,7 +433,7 @@ static ssize_t store_##file_name     \
  if (ret != 1)       \
   return -EINVAL;      \
          \
- ret = __cpufreq_set_policy(policy, &new_policy);  \
+ ret = cpufreq_set_policy(policy, &new_policy);  \
  policy->user_policy.object = policy->object;   \
          \
  return ret ? ret : count;     \
@@ -491,11 +491,7 @@ static ssize_t store_scaling_governor(struct cpufreq_policy *policy,
       &new_policy.governor))
   return -EINVAL;
 
- /*
-  * Do not use cpufreq_set_policy here or the user_policy.max
-  * will be wrongly overridden
-  */
- ret = __cpufreq_set_policy(policy, &new_policy);
+ ret = cpufreq_set_policy(policy, &new_policy);
 
  policy->user_policy.policy = policy->policy;
  policy->user_policy.governor = policy->governor;
@@ -837,11 +833,11 @@ static void cpufreq_init_policy(struct cpufreq_policy *policy)
  int ret = 0;
 
  memcpy(&new_policy, policy, sizeof(struct cpufreq_policy));
- /* assure that the starting sequence is run in __cpufreq_set_policy */
+ /* assure that the starting sequence is run in cpufreq_set_policy */
  policy->governor = NULL;
 
  /* set default policy */
- ret = __cpufreq_set_policy(policy, &new_policy);
+ ret = cpufreq_set_policy(policy, &new_policy);
  policy->user_policy.policy = policy->policy;
  policy->user_policy.governor = policy->governor;
 
@@ -1850,10 +1846,10 @@ int cpufreq_get_policy(struct cpufreq_policy *policy, unsigned int cpu)
 EXPORT_SYMBOL(cpufreq_get_policy);
 
 /*
- * data   : current policy.
- * policy : policy to be set.
+ * policy : current policy.
+ * new_policy: policy to be set.
  */
-static int __cpufreq_set_policy(struct cpufreq_policy *policy,
+static int cpufreq_set_policy(struct cpufreq_policy *policy,
     struct cpufreq_policy *new_policy)
 {
  int ret = 0, failed = 1;
@@ -2000,7 +1996,7 @@ int cpufreq_update_policy(unsigned int cpu)
   }
  }
 
- ret = __cpufreq_set_policy(policy, &new_policy);
+ ret = cpufreq_set_policy(policy, &new_policy);
 
  unlock_policy_rwsem_write(cpu);
 
-- 
1.7.1